app/lib/navigation/drawer/tiles/drawer_tiles.dart

```diff
@@ -22,6 +22,7 @@ const onlyDesktopTiles = <Widget>[
   DrawerTile(NavigationItem.homework),
   DrawerTile(NavigationItem.timetable),
   DrawerTile(NavigationItem.blackboard),
+  DrawerTile(NavigationItem.grades),
```

Comment:
If you merge the `main` into this branch, we can access the grades page with `Grades` as activation code.